src/components/organisms/NavigatorPane/NavigatorPane.tsx

```diff
@@ -18,16 +19,27 @@ import SectionRenderer from './SectionRenderer';
 import WarningsAndErrorsDisplay from './WarningsAndErrorsDisplay';
 import * as S from './NavigatorPane.styled';
 
-const NavPane = () => {
+const NavPane: React.FC = () => {
   const dispatch = useAppDispatch();
   const {windowSize} = useContext(AppContext);
   const windowHeight = windowSize.height;
   const navigatorHeight = windowHeight - NAVIGATOR_HEIGHT_OFFSET;
 
   const fileMap = useAppSelector(state => state.main.fileMap);
+  // TODO: remove this any
```

Comment:
Agreed, `any` can be replaced with `ResourceFilterType` from `models/appstate`

Reply:
Thanks for the advice!